css/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+
+    font-family: 'Poppins', sans-serif;
+
+}
+
+body {
+
+    background-image:linear-gradient(
+        115deg,
+        rgba(3, 227, 252, 0.8),
+        rgba(3, 20, 252, 0.8)
+      ), url('https://cdn.freecodecamp.org/testable-projects-fcc/images/survey-form-background.jpeg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: center;
+    /* background-color: rgba(0,0,0, 0.1); */
+
+    /* opacity: 0.6; */
```

Comment:
It would be best to remove unused code from your file rather than leave it in a comment in order to make your code cleaner

Reply:
Great job @lucascomamala 🤝

- [ ] Please make this comment human-friendly by adding emojis.👏